web/ts/watch.ts

```diff
@@ -1,12 +1,12 @@
 import { getPlayers } from "./TUMLiveVjs";
 import { copyToClipboard, Time } from "./global";
-import { seekbarOverlay } from "./seekbar-overlay";
```

Comment:
I assume this was unused?

Reply:
Yes (see beginning & end of [watch.ts file](https://github.com/TUM-Dev/gocast/blob/dev/web/ts/watch.ts))